src/test/bench_log.cc

```diff
@@ -30,8 +30,19 @@ struct T : public Thread {
   }
 };
 
+void usage(const char* program_name) {
+  cout << program_name << " <threads> <lines>" << std::endl;
+  cout << "\t threads: the number of threads for this test." << std::endl;
+  cout << "\t lines: the number of log entries per thread." << std::endl;
```

Comment:
nit, i'd suggest put something like
```c++
  cout << program_name << " <threads> <lines>"
       << "\t threads: the number of threads for this test."
       << "\t lines: the number of log entries per thread."
       << std::endl;
```

more compacted and concise this way.

Reply:
OK